runtime/tr.source/trj9/runtime/MetaData.cpp

```diff
@@ -1465,7 +1465,7 @@ createMethodMetaData(
 
    data->startPC = (UDATA)comp->cg()->getCodeStart();
    data->endPC = (UDATA)comp->cg()->getCodeEnd();
-   data->startColdPC = (UDATA)comp->cg()->getColdCodeStart();
+   data->startColdPC = (UDATA)0;
 
    if (data->startColdPC)
```

Comment:
i assume you'll fold this away in a subsequent commit? :)

Reply:
Yes, I'm doing this very conservatively, but I could fix this one up right away because its clearly dead.

Reply:
Fixed